### PR TITLE
[webgpu] Add program label for shader

### PR DESCRIPTION
**Pull request by axinging**
This is used for tracing, which helps us understand the mapping between program and createShaderModule.

To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/5905)

### CODE REVIEW

**Comment by axinging** [comment]
@qjia7 @haoyunfeix @xhcao @gyagp PTAL

File: tfjs-backend-webgpu/src/kernels/webgpu_program.ts [hunk 1]
```diff
@@ -73,7 +73,8 @@ export const compileProgram =
 
       const source = shader_preprocessor.makeShader(
           inputsData, outputData, program, isFromPixel);
-      const module = device.createShaderModule({code: source});
+      const module = device.createShaderModule(
+          {code: source, label: program.constructor.name});
       const pipeline = device.createComputePipeline(
           {layout: pipelineLayout, compute: {module, entryPoint: 'main'}});
```
Comment (qjia7) [inline tfjs-backend-webgpu/src/kernels/webgpu_program.ts:79]: label for `createComputePipeline` is also needed?

Comment (axinging) [reply tfjs-backend-webgpu/src/kernels/webgpu_program.ts]: Pass this twice seems wired. But It seems createComputePipeline doesn't get this label from shader module.
Should we change createComputePipeline in dawn?

Comment (qjia7) [reply tfjs-backend-webgpu/src/kernels/webgpu_program.ts]: That's a good point. Please file a bug to dawn to ask whether `createComputePipeline` can reuse `module`'s label if the label itself is not specified.

Comment (axinging) [reply tfjs-backend-webgpu/src/kernels/webgpu_program.ts]: Bug is here: https://bugs.chromium.org/p/dawn/issues/detail?id=1211

**Review by qjia7 — APPROVED** [review]

**Comment by qjia7** [comment]
@pyu10055 @lina128 @mattsoulanille Please help merge, thanks.

**Pull request merged**